File: sharedelementanimationsample/src/main/java/me/aartikov/sharedelementanimation/ui/SharedElementProvider.java
package me.aartikov.sharedelementanimation.ui;

import android.view.View;

import me.aartikov.alligator.AnimationData;

/**
 * Date: 16.04.2017
 * Time: 14:07
 *
 * @author Artur Artikov
 */
public interface SharedElementProvider {
	View getSharedElement(AnimationData animationData);

	String getSharedElementName(AnimationData animationData);
}

File: sharedelementanimationsample/src/main/java/me/aartikov/sharedelementanimation/ui/SharedElementProvider.kt
package me.aartikov.sharedelementanimation.ui;

import android.view.View;

import me.aartikov.alligator.animations.AnimationData;


public interface SharedElementProvider {
	View getSharedElement(AnimationData animationData);

	String getSharedElementName(AnimationData animationData);
}
